src/main/scala/cromwell/engine/CallActorState.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.engine
+
+sealed trait CallActorState {
+  def isTerminal: Boolean
```

Comment:
Since almost all of these are `false` you could save a few lines by making it false here and overriding it on the one which isn't